amm-executor/src/Spectrum/Executor/Backlog/Process.hs

```diff
@@ -0,0 +1,64 @@
+module Spectrum.Executor.Backlog.Process 
+  ( Backlog(..)
+  , mkBacklog
+  ) where
+
+import qualified Streamly.Prelude as S
+import Streamly.Prelude
+  ( IsStream )
+
+import Spectrum.Executor.Backlog.Service (BacklogService (BacklogService, put, drop))
+import Spectrum.Executor.EventSource.Stream
+
+import Control.Monad.IO.Class
+  ( MonadIO )
+import Control.Monad.Trans.Control
+  ( MonadBaseControl )
+import Control.Monad.Catch
+  ( MonadThrow )
+import Spectrum.Executor.Topic
+  ( ReadTopic(..) )
+import Spectrum.Common.Streaming.Class
+  ( Compile(drain) )
+import Spectrum.Executor.Data.OrderState (OrderInState, OrderState (Pending, Executed))
+import Prelude hiding (drop)
+
+newtype Backlog m = Backlog 
```

Comment:
I think it should not be `newtype` but `data`

Reply:
> @GusevTimofey Why keeping the wrapper when it can be eliminated? `newtype` is simply an optimization here.

Cause it is not a model but a service that holds some API

Reply:
@GusevTimofey why we should use "data" for services which contains one constructor with one field? We can use "newtype" for optimization